### ISSUE TITLE
Upgrade the universal forecast script of global-workflow for CCPP namelists, PR#32

### ISSUE THREAD

**Issue by jkuang-exploro** [NOAA-EMC/global-workflow#6]
The universal forecast script located in feature/coupled-crow branch need to be able to support CCPP namelists generation.

**Comment by jkuang-exploro** [comment]
@KateFriedman-NOAA I think this one is done. I'm working on a similar feature with Lin Gan for IPD namelists now and waiting for his comments.

**Comment by KateFriedman-NOAA** [comment]
Getting caught up on this work...where is this currently? Thanks!

**Comment by JessicaMeixner-NOAA** [comment]
I think right now we're still doing things in IPD, but we will need to make updates to the workflow when we migrate to CCPP.  I'm not 100% sure what the changes will be since we're still waiting on some code updates on the model side.  I will update this issue when I have more information.

**Comment by KateFriedman-NOAA** [comment]
Sounds good, thanks for the update!

**Comment by jkuang-exploro** [comment]
What is done: A script set that could support 2 CCPP suites(gsd, emc) and ipd runs
To be done: Make this script set working for feature/coupled-crow and merge it in.

**Comment by KateFriedman-NOAA** [comment]
@JianKuang-NOAA Checking on the status of this issue. Is this issue still needed or has the work for it been resolved and/or tracked in another issue? Thanks!

**Comment by jkuang-exploro** [comment]
@KateFriedman-NOAA PR#32 has already been merged, now working on PR#81. The goal of this PR is to fully integrate wave components into the workflow.

**Comment by KateFriedman-NOAA** [comment]
Gotcha, thanks for the update! Wanted to make sure this was still an active issue.

**Comment by KateFriedman-NOAA** [comment]
@JianKuang-NOAA Is the work for this issue complete? If so, please document completion of work in here and close issue. Thanks!

**Comment by jkuang-exploro** [comment]
@KateFriedman-NOAA This is completed long ago. PR#32 has already been merged. Please close this issue.

**Comment by KateFriedman-NOAA** [comment]
Closing, thanks @JianKuang-NOAA !
